From 4807b620f38a440c3ea0024f78bd4cd22d9463db Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Fri, 13 Oct 2017 19:42:24 +0300
Subject: [PATCH 130/767] staging: fsl-dpaa2/eth: Function rename

Rename dpaa2_eth_xdp_xmit() to dpaa2_eth_xdp_tx() to avoid
possible name conflicts with the (future) implementation of
ndo_xdp_xmit.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index 6af8e7e4f66e..359e049089b1 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -232,10 +232,10 @@ static struct sk_buff *build_frag_skb(struct dpaa2_eth_priv *priv,
 	return skb;
 }
 
-static int dpaa2_eth_xdp_xmit(struct dpaa2_eth_priv *priv,
-			      const struct dpaa2_fd *fd,
-			      void *buf_start,
-			      u16 queue_id)
+static int dpaa2_eth_xdp_tx(struct dpaa2_eth_priv *priv,
+			    const struct dpaa2_fd *fd,
+			    void *buf_start,
+			    u16 queue_id)
 {
 	struct dpaa2_eth_fq *fq;
 	struct rtnl_link_stats64 *percpu_stats;
@@ -360,8 +360,8 @@ static void dpaa2_eth_rx(struct dpaa2_eth_priv *priv,
 				release_fd_buf(priv, ch, addr);
 				goto drop_cnt;
 			case XDP_TX:
-				if (dpaa2_eth_xdp_xmit(priv, fd, vaddr,
-						       queue_id)) {
+				if (dpaa2_eth_xdp_tx(priv, fd, vaddr,
+						     queue_id)) {
 					dma_unmap_single(dev, addr,
 							 DPAA2_ETH_RX_BUF_SIZE,
 							 DMA_BIDIRECTIONAL);
-- 
2.17.0

